### ISSUE TITLE
FR: Add `--ignore-working-copy` hint to large file snapshot error 

### ISSUE THREAD

**Issue by fynnsu** [jj-vcs/jj#3912]
**Is your feature request related to a problem? Please describe.**
I recently ran into this issue when I temporarily added a large (>500 GB) untracked file to my repo. I wanted to run some tests, make a change to an upstream commit, switch back to my commit and run the tests again. Instead I found that I couldn't run any `jj` commands (even stuff like `jj log`) because it was trying to snapshot the large file.

My workaround was to add the file to `.gitignore` switch to the upstream commit, realize that I also need to change the upstream `.gitignore` just to switch back.

**Describe the solution you'd like**
While this doesn't fully solve my problem (since I can't record the change to the upstream commit), it would've been helpful to have `--ignore-working-copy` hinted as an option to allow me to run `jj log` and switch between commits. The current experience is that `jj` stops working entirely and its suggestions (increasing max file size to >500 GB or adding to `.gitignore`) aren't helpful (in my case).

**Additional context**
```
Error: Failed to snapshot the working copy
The file 'MY_LARGE_FILE' is too large to be snapshotted: it is 660.0MiB; the maximum size allowed is ~1.0MiB.
Hint: This is to prevent large files from being added on accident. You can fix this error by:
  - Adding the file to `.gitignore`
  - Run `jj config set --repo snapshot.max-new-file-size 692064256`
    This will increase the maximum file size allowed for new files, in this repository only.
  - Run `jj --config-toml 'snapshot.max-new-file-size=692064256' st`
    This will increase the maximum file size allowed for new files, for this command only.
```

**I think it would be helpful to add something like**
```
- Run the command with `--ignore-working-copy`
  This will ignore changes to the working copy that have not been previously snapshotted.
```

Then approaching this again, I would likely use `--ignore-working-copy` to list the commits, choose a commit high enough upstream, switch to the commit (again using `--ignore-working-copy`) and add the file to the `.gitignore` upstream.

**Comment by PhilipMetzger** [comment]
Ideally we shouldn't even fail in `jj st`, see https://github.com/martinvonz/jj/issues/3616#issuecomment-2096748218 but this seems to be nice improvement until that is done.

**Comment by cole-h** [comment]
Now that https://github.com/jj-vcs/jj/pull/5069 has merged (which changes the behavior to warn and continue instead of error when encountering large files), is this still desirable?

**Comment by thoughtpolice** [comment]
Good catch. I think this one can be closed pretty reasonably since the large file error is no longer fatal (and `--ignore-working-copy` didn't really help that much anyway.)
